content/dart/concepts/list/terms/join/join.md

```diff
@@ -0,0 +1,108 @@
+---
+Title: 'join' 
+Description: 'The join() method concatenates (combines) list elements as a string including various separators between them dependent on a user's need.'
+Subjects:
+  - 'Code Foundations'
+  - 'Web Development'
+Tags:
+  - 'Concatenation'
+  - 'Dart'
+  - 'Lists'
+CatalogContent:
+  - 'paths/computer-science'
+---
+
+The **`join()`** method concatenates (*combines*) list elements as a string including various separators between them dependent on a user's need.
+ 
+
+## Syntax
+
+```pseudo
+list-name.join(separator); // the raw syntax
+
+chores.join(" + "); // a real example
+```
+
+The method refers to its object, in the below case, to the 'chores' variable, which holds list (or array) elements. Inside the brackets '( )' there's a parameter - a separator containing any sign/-s wrapped with single/double quotes. The main role of a separator is to separate elements from one another - allowing for better readability and additional or new meaning to the text. Spaces added to a separator are nonobligatory.
+
+
+## Example 1
+
+The following block codes are showing an instance of removed spaces between elements, however a default `,` remains:
+
+```dart
+List<String> tought_to_read = ["A", "bit", "hard", "to", "read", "innit", "?"];
+
+String try_to_read = tought_to_read.join(); // Spaces between elements are removed.
+print(try_to_read); 
+```
+The output of `.join()`:

```

Comment:
```suggestion

The output of the above code is:

```
A line like this should be instead of the current one and remember to give the one-line space before and after a code block (Do this throughout the entire entry).